org_qubes_os_initial_setup/utils.py

```diff
@@ -50,6 +50,15 @@ def is_package_installed(pkgname):
     return bool(pkglist)
 
 
+def is_usb_device(device: pyudev.Device):
+    if device.get("ID_USB_INTERFACES", False):
+        return True
+    for parent in device.traverse():
```

Comment:
```suggestion
    for parent in device.ancestors:
```

Avoids introducing a new deprecation warning.

There's also one existing deprecation warning that can be avoided by changing `pyudev.Device.from_device_file` to `pyudev.Devices.from_device_file` (which still returns the same `Device` object).